Makefile

```diff
@@ -2,13 +2,18 @@ GOBIN ?= $$(go env GOPATH)/bin
 COVER_OUT ?= cover.out
 COVER_HTML ?= cover.html
 SCRIPTS_PATH ?= scripts
+BIN ?= bin
 
 E2E_ARTIFACTS_PATH ?= e2e/artifacts
 E2E_STATE_SETUP_PATH ?= e2e/optimism/.devnet
 E2E_CONFIG_SETUP_PATH ?= e2e/optimism/packages/contracts-bedrock/deploy-config/devnetL1.json
 FOUNDRY_ARTIFACTS_PATH ?= bindings/artifacts
 FOUNDRY_CACHE_PATH ?= bindings/cache
 
+.PHONY: monogen
+monogen:
+	go build -o $(BIN)/monogen ./monogen/cmd
```

Comment:
The ignite tool generates a bunch of comments used by starport scaffolding (e.g. `// this line is used by starport scaffolding # stargate/app/maccPerms`).

There are a few cases in the generated `testapp/app_config` file where the comment is overwritten by the monogen script. I don't know too much about how startport scaffolding works but do you know of any potential errors that could arise from overwriting that comment?

Reply:
Ohh overwriting the comment shouldn't happen. Possibly an oversight on my end--we're supposed to add the comment back. Will double check on my end.